packages/eslint-plugin/src/rules/prefer-readonly-parameter-types.ts

```diff
@@ -5,9 +5,11 @@ import * as util from '../util';
 
 type Options = [
   {
+    allowlist?: Array<util.TypeOrValueSpecifier>;
```

Comment:
> `allowlist`

I wonder if just `allow` would be better? The existing rule options are generally `allow*` where `*` is some description, such as `allowBoolean` or `Number`. What do you think?